### DISCUSSION TITLE
这个是特性还是bug？

### DISCUSSION THREAD

**Discussion by S-T-eve** [libnyanpasu/clash-nyanpasu#463] [General]
很久之前就发现个事，日志页的默认显示层级不是最高到info嘛
然后有天（就是之前GUI崩了但是core没退出那会）就心血来潮按了一下F5
然后就显示出debug层级的信息了
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/53549980/19eb0ce5-c0e6-431f-8ae9-60fd14c299f4)
所以这个算是“特性”还是bug

**Comment by S-T-eve** [comment]
哦，verge rev也这样，应该是特性
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/53549980/29255633-ed81-40f0-a0f8-8a4f59c3bfca)

**Reply by greenhat616** [reply]
正常情况下 f5 也会造成这样的问题？

**Reply by S-T-eve** [reply]
就正常用并且已经确认日志面板当前all时最高层级为info，在任何面板下F5再切到日志页就显示白字的debug层级了，Clash Verge Rev也是这样
另外刚刚发现完全关闭应用后重新正常打开是info，再右上角叉掉窗口再从托盘把窗口呼出来也会debug

**Comment by greenhat616** [comment]
感觉像是识别错误。你看日记文件里这个记录是 debug 么。



发自我的iPhone


------------------ 原始邮件 ------------------
From: STeveBlocks ***@***.***&gt;
发送时间: 02/20/2024, 18:25
To: LibNyanpasu/clash-nyanpasu ***@***.***&gt;
抄送: Jonson Petard ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [LibNyanpasu/clash-nyanpasu] 这个是特性还是bug？ (Discussion #463)





 
就正常用并且已经确认日志面板当前all时最高层级为info，在任何面板下F5再切到日志页就显示白字的debug层级了，Clash Verge Rev也是这样
 另外刚刚发现完全关闭应用后重新正常打开是info，再右上角叉掉窗口再从托盘把窗口呼出来也会debug

**Reply by S-T-eve** [reply]
是，内容能对上
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/53549980/a1196190-cc63-4998-8d19-ace3c9f49558)
但是和web ui上的比又不完全对上，不知道过滤掉了多少

**Reply by greenhat616** [reply]
看起来是 webui 的 bug。改天有空我调查一下。
